src/compile/layout.ts

```diff
@@ -61,7 +61,7 @@ function getCellHeight(model: Model): LayoutValue {
       return model.config().cell.height;
     }
   }
-  return model.fieldDef(Y).scale.bandWidth;
+  return 21 /* config.scale.bandWidth */;
```

Comment:
Oh, I'm aware that it's only 2 numbers but they are in 16 different places. Will you send a follow up PR?